README.md

```diff
@@ -71,17 +72,58 @@ make serve
 ### Short info about indexer multi-chain structure
 
 1. Chain configs/endpoints can be configured in `config.ts` file in appropriate folder of necessary chain:
-    - Subsocial - `src/chains/subsocial/config.ts`
-    - Soonsocial - `src/chains/soonsocial/config.ts`
+   - Subsocial - `src/chains/subsocial/config.ts`
+   - Soonsocial - `src/chains/soonsocial/config.ts`
 2. Necessary events/calls/storage calls can be configured for each chain in appropriate file in `/typegen` folder.
 3. Command `make typegen` uses custom [shel script](./scripts/typegen.sh) which generates types for each chain defined in `/typegen` folder.
-4. Chain sensitive logic is implemented in sub-folder with appropriate name in `/src/chains` folder. 
-It's required because each chain has it's own bunch of specs and as result it's own autogenerated types.
-Chain sensitive logic:
+4. Chain sensitive logic is implemented in sub-folder with appropriate name in `/src/chains` folder.
+   It's required because each chain has it's own bunch of specs and as result it's own autogenerated types.
+   Chain sensitive logic:
    - parsing events data
    - parsing calls data
    - making storage calls
 5. Indexer can be deployed to Aquarium to 2 different squids with 2 separate deployment manifests:
    - Subsocial - [./squid-subsocial.yaml](./squid-subsocial.yaml)
    - Soonsocial - [./squid-soonsocial.yaml](./squid-soonsocial.yaml)
-   
\ No newline at end of file
+
+## Search API
+
+Indexer GraphQL API has search query (`searchQuery`) with various parameters for filtering and full text search.
+
+### Search query arguments:
+
+- `indexes?: <string>` [ _default:_ `all` ] - _search within all or specific index (`all | spaces | posts`)_
+- `limit?: <number>` [ _default:_ `10` ] - _number of search hits per page._
+- `offset?: <number | null>` [ _default:_ `0` ] - _offset of search hits._
+- `q?: <string>` [ _default:_ `*` ] - _search query._
+- `spaceId?: <string>` - _filter search hits by provided `spaceId`._
+- `tags?: <string[]>` - _filter search hits by provided tags._
+
+All arguments listed above can be used together in any combination, except `spaceId + indexes:spaces`.
+
+### Search query results:
+
+- `err` - _search error if occurred. Can be `null` is not occurred._
```

Comment:
```suggestion
- `err` - _the search result if an error occurred. It can be `null` if no error occurred._
```
Not entirely sure about the 2nd sentence